zkvm/src/verifier.rs

```diff
@@ -4,10 +4,11 @@ use bulletproofs::{BulletproofGens, PedersenGens};
 use curve25519_dalek::ristretto::CompressedRistretto;
 use merlin::Transcript;
 use musig::{Multisignature, VerificationKey};
+use readerwriter::Encodable;
 
 use crate::constraints::Commitment;
 use crate::contract::ContractID;
-use crate::encoding::{Encodable, Reader};
+use crate::encoding::Reader;
```

Comment:
per suggestions above:

```suggestion
use crate::encoding::{Encodable, Reader};
```

Reply:
Ok.